tests/Unit/DataStructures/Tensor/EagerMath/CMakeLists.txt

```diff
@@ -11,6 +11,8 @@ set(LIBRARY_SOURCES
   Test_Magnitude.cpp
   Test_Norms.cpp
   Test_OrthonormalOneform.cpp
+  Test_OuterProduct.cpp
```

Comment:
This wasn't being built before?  Should be in a separate commit

Reply:
No apparently is wasn't 🤷‍♂️ I just noticed and added it